modules/mobile/spec/support/helpers/rails_helper.rb

```diff
@@ -17,7 +17,7 @@
     Flipper.enable('va_online_scheduling')
   end
 
-  config.after :each, :mobile_spec, type: :request do |example|
+  config.after :each, :mobile_spec, skipped?: false, type: :request do |example|
```

Comment:
We don't want to do this on skipped tests. It will cause warnings with error messages because `response` is nil.